domains/certificates/Query.API/API/MeasurementsSyncer/Metrics/MeasurementSyncMetrics.cs

```diff
@@ -19,6 +19,16 @@ public class MeasurementSyncMetrics : IMeasurementSyncMetrics
     private long recoveredMeasurements = 0;
     private ObservableCounter<long> RecoveredMeasurements { get; }
 
+    private long numberOfMeasurementsQuantityTooLow = 0;
+    private ObservableCounter<long> NumberOfMeasurementsQuantityTooLow { get; }
+
+    private long numberOfMeasurementsQuantityTooHigh = 0;
+    private ObservableCounter<long> NumberOfMeasurementsQuantityTooHigh { get; }
+
+    private long numberOfMeasurementsWrongQuality = 0;
+    private ObservableCounter<long> NumberOfMeasurementsWrongQuality { get; }
+
+
```

Comment:
kan vi få skrevet tests på at disse virker?

Reply:
As we talked about we dont have any test right now and i think it is a lot of work to test a `+=` statement